### PR TITLE
Move documentation build to circleci

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - store_artifacts:
+          path: docs/_build/html
+
+      - run:
+          name: "Built documentation is available at:"
+          command: DOCS_URL="${CIRCLE_BUILD_URL}/artifacts/${CIRCLE_NODE_INDEX}/${CIRCLE_WORKING_DIRECTORY/#\~/$HOME}/docs/_build/html/index.html"; echo $DOCS_URL
+
+notify:
+  webhooks:
+    - url: https://giles.cadair.com/circleci
```
Comment (nabobalis) [inline .circleci/config.yml]: What is this?

Comment (Cadair) [reply .circleci/config.yml]: Look at the Ci status! It provides a link to the documentation build.

Comment (nabobalis) [reply .circleci/config.yml]: OH this link provides that.

**Comment by Cadair** [comment]
unrelated online fails.

**Pull request merged**